contentcuration/contentcuration/tests/test_asynctask.py

```diff
@@ -0,0 +1,81 @@
+from base import BaseAPITestCase
+
+from contentcuration.models import Task
+from contentcuration.utils.asynctask import AsyncTask
+from contentcuration.tasks import create_async_task, non_async_test_task
+# from celery.contrib.testing.worker import start_worker
+# from contentcuration.celery import app
+
+
+class AsyncTaskTestCase(BaseAPITestCase):
```

Comment:
Woohoo tests! It would be nice to get an idea what it's testing, without reading the code first though. Can you add docstrings for each of these tests?

Reply:
Sure!